mtp_api/apps/transaction/tests/test_views.py

```diff
@@ -143,6 +146,25 @@ def test_fails_with_logged_in_managing_different_prison(self):
 
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
+    def test_fails_without_permissions(self):
+        """
+        Tests that logged-in user has to have the right permissions
+        to access the endpoint.
+        """
+        prison = self.prisons[0]
+        user = get_users_for_prison(prison)[0]
+
+        # delete user from groups
+        user.groups.all().delete()
+
+        url = self._get_list_url(prison)
+
+        self.client.force_authenticate(user=user)
+
+        response = self.client.get(url, format='json')
+
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
```

Comment:
As discussed, it'd be nice to reduce some of the duplication, maybe with a mixin or something.